sdk/include/reactos/mc/netmsgmsg.mc

```diff
@@ -12951,7 +12951,7 @@ Language=English
 The Alerter service had a problem creating the list of\nalert recipients.  The error code is %1.
 .
 Language=Polish
-The Alerter service had a problem creating the list of\nalert recipients.  The error code is %1.
+Usługa Alertowanie napotkała na problem przy tworzeniu\nlisty odbiorców alertu. Kod błędu to %1.
```

Comment:
`Usługa alarmowania Alerter napotkała na problem przy tworzeniu\nlisty odbiorców alertu. Kod błędu to %1.`
From Win XP SP3